content/ADR-110-realm-description.md

```diff
@@ -61,51 +63,123 @@ type About = {
 }
 ```
 
-Example:
+Example https://peer.decentraland.org/about:
 
 ```json
 {
   "healthy": true,
   "content": {
     "healthy": true,
-    "version": "5.1.2",
-    "commitHash": "dce61c002c89db4966c0cdd008d3d654f297050b",
-    "publicUrl": "https://peer.decentraland.org/content"
+    "version": "6.0.6",
+    "commitHash": "d2eeccaffe2a9c22ac963348851c7791b63fc517",
+    "publicUrl": "https://peer-ec2.decentraland.org/content/"
+  },
+  "lambdas": {
+    "healthy": true,
+    "version": "6.0.6",
+    "commitHash": "d2eeccaffe2a9c22ac963348851c7791b63fc517",
+    "publicUrl": "https://peer-ec2.decentraland.org/lambdas/"
   },
   "configurations": {
-    "realmName": "catalyst-4",
-    "networkId": 1
+    "networkId": 1,
+    "globalScenesUrn": [],
+    "scenesUrn": [],
+    "realmName": "hela"
   },
   "comms": {
     "healthy": true,
     "protocol": "v3",
-    "fixedAdapter": "ws-room:mini-comms.decentraland.org/rooms/test-room"
+    "commitHash": "b3ec3327ceef53473853068dcdad8ea08d7a0f9c"
+  },
+  "bff": {
+    "healthy": true,
+    "commitHash": "1a2ff915a216191ecc6ef85f3822f0809fe16f3c",
+    "userCount": 4,
+    "protocolVersion": "1.0_0",
+    "publicUrl": "/bff"
+  },
+  "acceptingUsers": true
+}
+```
+
+Example https://worlds-content-server.decentraland.org/world/menduz.dcl.eth/about:
+
+```json
+{
+  "healthy": true,
+  "configurations": {
+    "networkId": 1,
+    "globalScenesUrn": [],
+    "scenesUrn": [
+      "urn:decentraland:entity:bafkreihiv5zkzjui46gvxtsnk5pfogmq7kyzijxpf3gjqlb2ivydcuwgxq?baseUrl=https://worlds-content-server.decentraland.org/contents/"
+    ],
+    "minimap": {
+      "enabled": false
+    },
+    "skybox": {},
+    "realmName": "menduz.dcl.eth"
+  },
+  "content": {
+    "healthy": true,
+    "publicUrl": "https://peer.decentraland.org/content"
   },
   "lambdas": {
     "healthy": true,
-    "version": "5.1.2",
-    "commitHash": "dce61c002c89db4966c0cdd008d3d654f297050b",
     "publicUrl": "https://peer.decentraland.org/lambdas"
   },
-  "bff": {
+  "comms": {
     "healthy": true,
-    "commitHash": "369c5dafeda62a1b16f5232cd477565cc3f3d513",
-    "userCount": 0,
-    "publicUrl": "/"
-  }
+    "protocol": "v3",
+    "fixedAdapter": "signed-login:https://worlds-content-server.decentraland.org/get-comms-adapter/world-prd-menduz.dcl.eth"
+  },
+  "acceptingUsers": true
 }
 ```
 
-## Health
+## Health property
 
-If the realm is in healthy state, the endpoint must return a http status 200, otherwise it should return 503. This way just by checking the response http status, a client looking for a realm to connect may decide to connect or not to the realm.
+`healthy: boolean` is used to inform whether the realm is in healthy state, the endpoint must return a http status 200, otherwise it should return 503. This way just by checking the response http status, a client looking for a realm to connect may decide to connect or not to the realm.
```

Comment:
```suggestion
`healthy: boolean` is used to inform whether the realm is in a healthy state, the endpoint must return an HTTP status of 200, otherwise it should return 503. This way just by checking the response HTTP status, a client looking for a realm to connect may decide whether to join or not the realm.
```